#Node
int64 node
float64 x_pose
float64 y_pose
float64 z_pose
float64 yaw
